### DISCUSSION TITLE
디스코드의 새로운 테마색에 관하여

### DISCUSSION THREAD

**Discussion by wonderlandpark** [koreanbots/koreanbots#363] [일반] [answered]
디스코드의 Rebrand 업데이트로, 디스코드 로고를 비롯한 테마색 등이 변경되었습니다.

이에 따라, 사이트에서 사용중이던 기존 Blurple(#7289DA)을 새로운 Blurple(#5865F2)로 변경하여야할지, 아니면 일부만 변경할지 등에 대하여 토론하고자합니다.

v2를 예시로 색을 바꿔보겠습니다.
**초대하기 버튼**
기존
<img width="345" alt="image" src="https://user-images.githubusercontent.com/31924512/118400997-ea7bc500-b69e-11eb-8117-f31675bcc533.png">

새로운 색
<img width="325" alt="image" src="https://user-images.githubusercontent.com/31924512/118401035-0da67480-b69f-11eb-9da1-2a7f988ef693.png">

**디스코드 인증됨 아이콘**
기존
<img width="207" alt="image" src="https://user-images.githubusercontent.com/31924512/118401094-41819a00-b69f-11eb-93ad-2dba47916da2.png">

새로운 색
<img width="208" alt="image" src="https://user-images.githubusercontent.com/31924512/118401083-3a5a8c00-b69f-11eb-9bd5-408a732f27d7.png">

**Comment by wonderlandpark** [comment]
추가적으로 디스코드 아이콘도 새로운 아이콘으로 변경해야할지, 유지해도 무방할지도 의견을 받고 싶습니다
<img width="344" alt="image" src="https://user-images.githubusercontent.com/31924512/118401171-8c9bad00-b69f-11eb-9ba8-4b5bbde8154d.png">

**Reply by minibox24** [reply]
아래에 제가 남긴 의견과 마찬가지로 독자적인 아이콘이 아닌 디스코드의 아이콘이기에 디스코드의 아이콘이 변경된다면 여기 있는 아이콘도 바뀌어야 할 것 같습니다

**Comment by ghost** [comment]
인증됨은 아직 디코에서도 바뀌지 않았고, 기존 색깔을 유지하는게 더 좋을 것 같네요

**Reply by wonderlandpark** [reply]
<img width="249" alt="image" src="https://user-images.githubusercontent.com/31924512/118401270-ffa52380-b69f-11eb-93d6-419ff9f15b83.png">
디스코드에서도 변경되었습니다.

**Reply by ghost** [reply]
그러면 바꿔야죠

**Comment by minibox24** [comment]
디스코드의 Blurple는 모두 리브랜딩된 새로운 색으로 바꾸는게 맞다고 봅니다. 지금 적용되어있는 Blurple는 독자적인 색이 아닌 디스코드의 색을 가져온것이기에 디스코드의 테마 색이 바뀌면 한디리도 이에 맞춰 따라가야 한다고 생각합니다.

**Comment by wonderlandpark** [comment, marked as answer]
새로운 테마색을 적용했습니다. 의견 남겨주셔서 감사드립니다!
